tests/src/test/java/alluxio/client/fs/LoadMetadataIntegrationTest.java

```diff
@@ -187,6 +192,29 @@ public void loadNeverConfiguration() throws Exception {
     checkGetStatus("/mnt/dir1/dirA/fileDNE1", options, false, false);
   }
 
+  @Test
+  public void loadRecursive() throws Exception {
+    Configuration.set(PropertyKey.USER_FILE_METADATA_LOAD_TYPE, LoadMetadataType.Once.toString());
+    ListStatusOptions options = ListStatusOptions.defaults().setRecursive(true);
+    for (int i = 0; i < 10; i++) {
+      FileWriter fileWriter = new FileWriter(mLocalUfsPath + "/dir1/file" + i);
+      fileWriter.write("test" + i);
+      fileWriter.close();
+    }
+
+    for (int i = 0; i < 10; i++) {
```

Comment:
can you make the structure with more directories (not just more files)? Maybe something like:

```
/dir1/dirA/file
/dir1/dirB/file
/dir1/dirC/file
...
/dir2/dirA/file
/dir2/dirB/file
...
```

Reply:
done.